webpack/localization-plugin/README.md

```diff
@@ -0,0 +1,181 @@
+
+##### `localizedData.passthroughLocale.usePassthroughLocale = true | false`
+
+If `passthroughLocale.usePassthroughLocale` is set to `true`, a passthrough locale will be included in the output.
+By default, the passthrough locale's name is "passthrough."
+
+##### `localizedData.passthroughLocale.passthroughLocaleName = '...'`
+
+If `passthroughLocale.usePassthroughLocale` is set to `true`, the "passthrough" locale name can be overridden
+by setting a value on `passthroughLocale.passthroughLocaleName`.
+
+#### `localizedData.pseudolocales = { }`
+
+This option allows pseudolocales to be generated from the strings in the default locale. This option takes
+an option with pseudolocales as keys and options for the
+[pseudolocale package](https://www.npmjs.com/package/pseudolocale) as values.
+
+### `filesToIgnore = [ ]`
+
+This option is used to specify `.resx` and `.loc.json` files that should not be processed by this plugin.
+By default, every `.resx` and `.loc.json` file import is intercepted by this plugin, and an error occurs
+if translations aren't provided for an intercepted file. To avoid that error, list files that should be ignored
+by this plugin in this property. Files should be specified as either absolute paths or paths relative
+to the Webpack compilation context.
+
+### `localizationStats = { }`
+
+#### `localizationStats.dropPath = '...'`
+
+This option is used to designate a path at which a JSON file describing the localized assets produced should be
+written. If this property is omitted, the stats file won't be written.
+
+The file has the following format:
+
+```JSON
+{
+  "entrypoints": {
+    "<BUNDLE NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    },
+    "<BUNDLE NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    }
+  },
+  "namedChunkGroups": {
+    "<CHUNK NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    },
+    "<CHUNK NAME>": {
+      "localizedAssets": {
+        "<LOCALE NAME>": "<ASSET NAME>",
+        "<LOCALE NAME>": "<ASSET NAME>"
+      }
+    }
+  }
+}
+
+```
+
+#### `localizationStats.callback = (stats) => { ... }`
+
+This option is used to specify a callback to be called with the stats data that would be dropped at
+[`localizationStats.dropPath`](#localizationStats.DropPath--) after compilation completes.
+
+### `typingsOptions = { }`
+
+This option is used to specify how and if TypeScript typings should be generated for loc files.
+
+It takes two options:
+
+#### `typingsOptions.generatedTsFolder = '...'`
+
+This property specifies the folder in which `.d.ts` files for loc files should be dropped. It is recommended
+that this be a folder parallel to the source folder, specified in addition to the source folder in the
```

Comment:
If we want to encourage adoption, it might be useful to provide a small example project that illustrates how to use the plugin. Like maybe a "Hello, world" React app, that depends on a library package exporting a Button control.  Then we could illustrate how webpack bundles the two projects together, and show how when the app switches between English/Spanish, the Button caption gets localized as well.  This would also illustrate how the typings work.

The demo could be really simple and use a barebones toolchain like from the TSDoc repo.  But this might clarify how all the pieces work together, what the JSON files look like, etc.

Reply:
Good call. I'll link to the example projects in this repo.